src/main/java/bookstore/controller/BookController.java

```diff
@@ -24,33 +25,37 @@
 @Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping(value = "/api/books")
+@RequestMapping(value = "/books")
 public class BookController {
     private final BookService bookService;
 
     @Operation(summary = "Get all books", description = "Get a list of all available books")
     @GetMapping
+    @PreAuthorize("hasRole('USER')")
```

Comment:
do you think ADMIN should not be allowed to use this end point?

Reply:
We have N:N connection User:Role. So, if user with Admin role wants to get info from books, role User should be ordered. Furthermore after discussing, made a decision to give a default role User when while a new User creating.